torch/_inductor/graph.py

```diff
@@ -876,15 +881,15 @@ def run_node(self, n: torch.fx.Node):
 
         return result
 
-    def check_cpp_codegen_disabled(self):
+    def validate_can_generate_cpp_wrapper(self):
         if config.disable_cpp_codegen:
-            self.disable_cpp_wrapper("cpp codegen disabled")
+            metrics.disable_cpp_wrapper += 1
```

Comment:
We don't really need this anymore as we fail fast now?

Reply:
Yeah, I was not sure about this. Let me remove it.